examples/quickstart-pandas/pandas_example/server_app.py

```diff
@@ -1,90 +1,55 @@
 """pandas_example: A Flower / Pandas app."""
 
-from typing import Dict, List, Optional, Tuple, Union
-
+from logging import INFO
 import numpy as np
-
+import flwr as fl
 from flwr.common import (
     Context,
-    EvaluateIns,
-    EvaluateRes,
-    FitIns,
-    FitRes,
-    Parameters,
-    Scalar,
-    ndarrays_to_parameters,
-    parameters_to_ndarrays,
+    MessageType,
+    RecordSet,
 )
-from flwr.server import ServerApp, ServerAppComponents, ServerConfig
-from flwr.server.client_manager import ClientManager
-from flwr.server.client_proxy import ClientProxy
-from flwr.server.strategy import Strategy
-
-
-class FedAnalytics(Strategy):
-    def initialize_parameters(
-        self, client_manager: Optional[ClientManager] = None
-    ) -> Optional[Parameters]:
-        return None
-
-    def configure_fit(
-        self, server_round: int, parameters: Parameters, client_manager: ClientManager
-    ) -> List[Tuple[ClientProxy, FitIns]]:
-        config = {}
-        fit_ins = FitIns(parameters, config)
-        clients = client_manager.sample(num_clients=2, min_num_clients=2)
-        return [(client, fit_ins) for client in clients]
+from flwr.common.logger import log
+from flwr.server import Driver
```

Comment:
Let's also import `ServerApp` here. Then, we can remove the `import flwr as fl` line and use `app = ServerApp()` below (like other examples).

Reply:
yes, let's do that